src/components/LogosSection/BackedBy/index.js

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+import { FormattedMessage } from "gatsby-plugin-intl"
+import { useStaticQuery, graphql } from "gatsby"
+import Img from "gatsby-image"
+
+import {
+  imageContainer,
+  backedByImage,
+  inlineContainer,
+  inlineContainer1,
+  andMore,
+} from "./style"
+
+const BackedBy = () => {
+  const Image = ({ fluid }) => <Img css={backedByImage} fluid={fluid} />
```

Comment:
Please put this outside of `<BackedBy />`

Reply:
done.